### PR TITLE
updated two defaults

### PR DESCRIPTION
**Pull request by edwardmedia**
Fixes https://github.com/hashicorp/terraform-provider-google/issues/12411

If this PR is for Terraform, I acknowledge that I have:

- [x] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [x] [Generated Terraform](https://github.com/GoogleCloudPlatform/magic-modules#generating-the-terraform-providers), and ran `make test` and `make lint` to ensure it passes unit and linter tests.
- [x] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/third_party/terraform/tests) (for handwritten resources or update tests).
- [x] [Ran](https://github.com/hashicorp/terraform-provider-google/blob/main/.github/CONTRIBUTING.md#tests) relevant acceptance tests (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [x] Read the [Release Notes Guide](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/.ci/RELEASE_NOTES_GUIDE.md) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:bug
vpcaccess: updated defaults for `minThroughput` and `maxThroughput` on `google_vpc_access_connector`
```

### CODE REVIEW

**Comment by edwardmedia** [comment]
@melinath Is it proper to update the valid range from 200-1000 to 500-1000 for the validation function here? What will happen to the existing ones below 500? Happy to remove the proposed change for the function to keep as is. Please let me know

**Comment by edwardmedia** [comment]
@melinath noticed that there is an existing validation function which currently allows the range from 200-1000. Due to the backward compatibility concern, decided to keep it as is (instead of change the range to 500-1000

**Comment by melinath** [comment]
This would be a breaking change for existing users - unless they're already broken in all cases by using this default?

Alternately could we change these fields to default_from_api?

**Comment by edwardmedia** [comment]
added `default_from_api`

File: mmv1/products/vpcaccess/api.yaml [hunk 1]
```diff
@@ -102,9 +102,10 @@ objects:
         min_version: beta
       - !ruby/object:Api::Type::Integer
         name: minThroughput
+        # its default in API doc is 200 but api returns 500 if it is not set
         description: |
-          Minimum throughput of the connector in Mbps. Default and min is 200.
-        default_value: 200
+          Minimum throughput of the connector in Mbps. Default and min is 500.
+        default_value: 500
```
Comment (melinath) [inline mmv1/products/vpcaccess/api.yaml:108]: We need to remove the defaults in addition to setting optional + computed. It's fine to update the docstrings though.

```suggestion
```

File: mmv1/products/vpcaccess/api.yaml [hunk 2]
```diff
@@ -117,12 +118,9 @@ objects:
...
       - !ruby/object:Api::Type::Integer
         name: maxThroughput
-        # The API documentation says this will default to 200, but when I tried that I got an error that the minimum
-        # throughput must be lower than the maximum. The console defaults to 1000, so I changed it to that.
-        # API returns 300 if it is not sent
         description: |
-          Maximum throughput of the connector in Mbps, must be greater than `min_throughput`. Default is 300.
-        default_value: 300
+          Maximum throughput of the connector in Mbps, must be greater than `min_throughput`. Default is 1000.
+        default_value: 1000
```
Comment (melinath) [inline mmv1/products/vpcaccess/api.yaml:123]:
```suggestion
```

**Review by melinath — CHANGES_REQUESTED** [review]

**Review by melinath — APPROVED** [review]
LGTM

**Pull request merged**